src/functions/updateTechRecords.ts

```diff
@@ -2,16 +2,17 @@ import TechRecordsDAO from "../models/TechRecordsDAO";
 import TechRecordsService from "../services/TechRecordsService";
 import HTTPResponse from "../models/HTTPResponse";
 import ITechRecordWrapper from "../../@Types/ITechRecordWrapper";
+import {ONLY_DIGITS_AND_NUMBERS} from "../assets/Enums";
 
 const updateTechRecords = (event: any) => {
   const techRecordsDAO = new TechRecordsDAO();
   const techRecordsService = new TechRecordsService(techRecordsDAO);
-  const ONLY_DIGITS_AND_NUMBERS: RegExp = /^[A-Za-z0-9]+$/;
 
   const techRec = event.body ? event.body.techRecord : null;
   const msUserDetails = event.body ? event.body.msUserDetails : null;
   const vin = event.pathParameters ? event.pathParameters.vin : null;
   const systemNumber = event.body ? event.body.systemNumber : null;
+  const newStatusCode = event.pathParameters ? event.body.newStatusCode : null;
```

Comment:
you're checking if pathParameters is defined and then retrieve newStatusCode from the body. newStatusCode should be a query param, so you should do something like 
const newStatusCode = event.queryStringParameters ? event.queryStringParameters.newStatusCode : null
Also, I think pathParameters will always be present on the event, so no need to check if it's defined